platformio.ini

```diff
@@ -135,3 +135,7 @@ extends = oled_common
 build_flags = 
   ${common.build_flags}
   -D MAIN_HW_PIN=19
+
+[env:ESP32C3]
+extends = oled_common
+board = ttgo-t-oi-plus
```

Comment:
please remove it, because we already have a env to this board

Reply:
Done